src/utils/confighandler.cpp

```diff
@@ -120,6 +120,7 @@ static QMap<class QString, QSharedPointer<ValueHandler>>
     OPTION("predefinedColorPaletteLarge", Bool               ( PREDEFINED_COLOR_PALETTE_LARGE )),
     // NOTE: If another tool size is added besides drawThickness and
     // drawFontSize, remember to update ConfigHandler::toolSize
+    OPTION("lastRegion"                  ,String             ( ""            )),
```

Comment:
I am utterly opposed to the inclusion of volatile data into the config. I quote what @mmahmoudian said in https://github.com/flameshot-org/flameshot/issues/1740#issuecomment-896567222 
> Many do version control their dotfiles and I personally hate it when a software change the dotfile without my permission and knowledge (e.g [Konsole](https://konsole.kde.org/), [screenkey](https://gitlab.com/screenkey/screenkey/)) because it will ruin the `git status`.

I propose we add a `volatile.ini` as a sibling of `flameshot.ini` so that the user can still change this value manually if needed. Or you can put it in `~/.local/yadda-yadda`, but I find this to be sub-optimal.

Reply:
Hmmmm, if we move this to another file it probably won't be ready for V11. I'm okay with that either way. 

Maybe we can just use https://doc.qt.io/qt-5/qtemporarydir.html#details, the downside is it will not preserve the last screenshot between invocations.

Reply:
@borgmanJeremy I feel that such a constraint would be felt by the users.

Reply:
Then lets use: QStandardPaths::CacheLocation

I think on linux that will be ~/.cache which also conforms to the XDG specification